src/include/daos/tests_lib.h

```diff
@@ -42,6 +42,12 @@
 	if (__rc != 0) \
 		fail_msg("Not successful!! Error code: " DF_RC, DP_RC(__rc)); \
 	} while (0)
+#define assert_rc_equal(rc, expected_rc) do { \
+	if((rc) == (expected_rc)) \
```

Comment:
(style)  space required before the open parenthesis '('

Reply:
add space between if and ( to fix this one

I also prefer just a \ on the define line so the do is lined up with while